src/XMakeBuildEngine/Resources/Strings.resx

```diff
@@ -1161,6 +1161,10 @@
     <value>MSB4173: The element &lt;{0}&gt; beneath element &lt;{1}&gt; is invalid because a child element with that name already exists</value>
     <comment>{StrBegin="MSB4173: "}</comment>
   </data>
+  <data name="InvalidAttributeDueToDuplication" UESanitized="false" Visibility="Public">
+    <value>MSB4227: The attribute &lt;{0}&gt; on element &lt;{1}&gt; is invalid because an attribute with that name already exists</value>
```

Comment:
I'm matching the formatting here from the previous error message for a similar situation